packages/calcite-components/src/components/notice/notice.tsx

```diff
@@ -228,17 +210,7 @@ export class Notice
   @Method()
   async setFocus(): Promise<void> {
     await componentFocusable(this);
-
-    const noticeLinkEl = this.el.querySelector("calcite-link");
```

Comment:
Can we split up the `setFocus` refactoring?

Reply:
Yeah I can revert this. 👍